src/app/code/community/SchumacherFM/OpCachePanel/Model/Types/OpCache.php

```diff
@@ -60,6 +61,23 @@ protected function _getCompiledFiles()
         return $this->_compiledFiles;
     }
 
+    
+    /**
+     * @return array
+     */
+    protected function _getBlackList()
+    {
+        if (NULL !== $this->_blackList) {
+            return $this->_blackList;
+        }
+
+        $config = opcache_get_configuration();
+
+        $this->_getBlackList = $config['blacklist'];
```

Comment:
use http://php.net/manual/en/function.array-flip.php

$this->_blackList = array_flip($config['blacklist'])

you will avoid lots of iteration during the comparison phase where I suggest using `isset`